event-sourcing/event-source-api-integration-test/conf/conf.d/cxf-rs.properties

```diff
@@ -0,0 +1,3 @@
+server      = org.apache.openejb.server.cxf.rs.CxfRSService
```

Comment:
Not sue these files in "conf" should be part of the commit... shouldn't they be excluded via .gitignore?

Reply:
I'll add them

Reply:
The files themselves are still in the commit, though, and need removing